File: framework/ui-elements/src/displays/components/Stamp.js
import React from "react";
import { withStyles } from '@material-ui/core/styles';
import AbstractStamp from "../../../gen/displays/components/AbstractStamp";
import StampNotifier from "../../../gen/displays/notifiers/StampNotifier";
import StampRequester from "../../../gen/displays/requesters/StampRequester";
import DisplayFactory from 'alexandria-ui-elements/src/displays/DisplayFactory';

const styles = theme => ({});

class Stamp extends AbstractStamp {

	constructor(props) {
		super(props);
		this.notifier = new StampNotifier(this);
		this.requester = new StampRequester(this);
	};


}

export default withStyles(styles, { withTheme: true })(Stamp);
DisplayFactory.register("Stamp", withStyles(styles, { withTheme: true })(Stamp));

File: framework/ui/src/io/intino/alexandria/ui/displays/components/Stamp.java
package io.intino.alexandria.ui.displays.components;

import io.intino.alexandria.core.Box;
import io.intino.alexandria.ui.displays.notifiers.StampNotifier;

public class Stamp<DN extends StampNotifier, B extends Box> extends AbstractStamp<B> {

    public Stamp(B box) {
        super(box);
    }

}